CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+### citus v9.5.11 (April 20, 2023) ###
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table (#5458)
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.0.7 (April 20, 2023) ###
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes columnar freezing/wraparound bug (#5962)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.1.5 (April 20, 2023) ###
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker (#5679)
+
+* Fixes columnar freezing/wraparound bug (#5962)
+
+* Fixes memory leak issue with query results that returns single row (#6724)
+
+* Prevents alter table functions from dropping extensions (#5974)
+
+### citus v10.2.9 (April 20, 2023) ###
```

Comment:
I have problems coming up with changelog items for a single commit on release-10.2 branch.

1. https://github.com/citusdata/citus/pull/6624 is cherry-picked to release-11.1 branch. This PR is shipped with 11.2 release but the changelog for that version do not seem to contain a similar entry that is supplied in the PR description. @onurctirtir This one is covered in 2 of the other backports. If you answered one of them, I guess the same will apply here as well.